### DISCUSSION TITLE
Cedel CEMM

### DISCUSSION THREAD

**Discussion by muller119** [home-assistant/frontend#9736] [Other feature requests] [answered]
can you make a intergration for the CEDEL CEMM dsmr P1 reader?
it has local API read out per 5sec
it can be used for the new energy frontend.

the dev website is http://developer.cemm.nl/
and the normal site is https://cedel.nl/

if you have any quostions please contact me.

**Comment by spacegaier** [comment, marked as answer]
This would be a backend/core topic, nothing for the frontend. But you either have to create the integration yourself, or wait for someone using that reader to come along and create one (or reach out to the vendor to see if they might be willing to provide an integration).
